guides/building-helm-charts-for-spk.md

```diff
@@ -0,0 +1,257 @@
+# Building Helm Charts for SPK and Bedrock
+
+### Prerequsities
+
+This guide assumes that you are familiar with understanding and creating
+[helm charts](https://helm.sh) - which are consumed by
+[fabrikate](https://github.com/microsoft/fabrikate) and `spk`.
+
+It also assumes that you have set up your `spk` project, and installed all
+necessary pipelines created via `spk project init` (the lifecycle pipeline),
+`spk service create` (build update hld pipeline), and `spk hld init` (manifest
+generation pipeline).
+
+### Introduction
+
+Recall the three git repositories that exist, and the pipelines that exist
+between them:
+
+![Repositories and pipelines](./images/spk-resource-diagram.png)
+
+In this guide, we assume a single application repository with a single service,
```

Comment:
Is this true because the diagram shows a monorepo?

Reply:
yep!